src/_scss/pages/modals/aboutTheData/_aboutTheData.scss

```diff
@@ -147,6 +174,10 @@
                 text-align: right;
                 padding-right: rem(42);
               }
+              .usda-table__cell:last-child {
```

Comment:
We do have a right-align property for the `Table` component that can be specified when you pass the columns. Example [here](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/containers/aboutTheData/AgencyDetailsContainer.jsx#L53)

Reply:
Thanks, I definitely did not know that. I updated the css and add the right property to the appropriate columns and it threw the css off. I can demo the changes with the right property if you like.